src/lib/copy.js

```diff
@@ -0,0 +1,21 @@
+if ('clipboard' in navigator && 'writeText' in navigator.clipboard) {
+  document.addEventListener('click', async (event) => {
+    if (
```

Comment:
To avoid the typing errors, this should be:

```js
if (!(event.target instanceof Element) || !event.target.classList.contains('w-headline-link')) {
  return;
}
```

Reply:
Good idea. Done.